docs/sphinx/users/fiji/index.rst

```diff
@@ -38,7 +38,7 @@ an earlier version using this guide if you need to.
 4) Remove bio-formats_plugins.jar from the 'plugins' sub-folder
 5) Remove all of the .jars from the 'jars/bio-formats' sub-folder:
 
-   - jai_imageio.jar
+   - ome-jai.jar
```

Comment:
`ome-codecs.jar` should probably be listed here

With decoupling, performing this operation manually will become increasingly burdensome. Instead it should be possible to use https://github.com/openmicroscopy/bio-formats-fiji directly. Updating these instructions to consume this repo is outside the scope of this PR.